core/spec/models/spree/variant_spec.rb

```diff
@@ -520,4 +520,28 @@
       expect(variant.dimension).to eq (dimension_expected)
     end
   end
+
+  context '#discontinue!' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.